syntax = "proto3";

package request_models;

message ClientCredentialsRequestModel {
	int32 player_id = 1;
	string username = 2;
	string token = 3;
}